lib/board.rb

```diff
@@ -0,0 +1,24 @@
+class Board
+  attr_accessor :positions
```

Comment:
You should not be exposing classes internal data by default. Think carefully about how other classes will interact with this one and implement proper methods around these usages. Rarely exposing the full internal data is necessary.